PyInstaller/lib/modulegraph/modulegraph.py

```diff
@@ -2581,14 +2581,11 @@ def _scan_bytecode(
               parsing will have already parsed import statements, which this
               parsing must avoid repeating.
         """
-        constants = module_code_object.co_consts
-
         level = None
         fromlist = None
 
-        prev_insts = []
-
-        for inst in get_instructions(module_code_object):
+        prev_insts = deque(maxlen=2)
+        for inst in util.enumerate_instructions(module_code_object):
```

Comment:
Would you mind splitting this commit int two? The `deque` change is separate and it is also important. Remember that I completely missed this change when you first proposed it?